hash/vis.xml

```diff
@@ -0,0 +1,1265 @@
+		<rom name="Kids Can Read! Cinderella - The Original Fairy Tale (USA) (En,Es).cue" size="131" crc="6e0f6066" sha1="84b7b512c7bd7fb07bc5602890bf7fa2aa605049"/>
+		-->
+		<description>Kids Can Read! Cinderella - The Original Fairy Tale</description>
+		<year>1992</year>
+		<publisher>Discis Knowledge Research</publisher>
+		<info name="serial" value="25-6131 / CDRM1024480"/>
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Kids Can Read Cinderella - The Original Fairy Tale (USA) (En,Es)" sha1="d74cd19296d8007ce93f97a66d4775ddbb449db0"/>
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="kids_homerun">
+		<!--
+		Origin: redump.org
+		<rom name="Kids Can Read! Heather Hits Her First Home Run by Ted Planos (USA) (En,Es).bin" size="317520000" crc="acdd851f" sha1="49174711e4f19a5ef2fed5d02df18b3edf14c028"/>
+		<rom name="Kids Can Read! Heather Hits Her First Home Run by Ted Planos (USA) (En,Es).cue" size="140" crc="a9be62c4" sha1="b47c7107c29488a8bb0c91a141570e405635d107"/>
+		-->
+		<description>Kids Can Read! Heather Hits Her First Home Run by Ted Planos</description>
+		<year>1992</year>
+		<publisher>Discis Knowledge Research</publisher>
+		<info name="serial" value="25-6133 / CDRM1024500"/>
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Kids Can Read Heather Hits Her First Home Run by Ted Planos (USA) (En,Es)" sha1="ef3d050d90686f4f21d3a74acf506094557146a5"/>
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="kids_nightbc">
+		<!--
+		Origin: redump.org
+		<rom name="Kids Can Read! The Night Before Christmas - Clement C. Moore LLD (USA) (En,Es).bin" size="289197216" crc="f87ea0be" sha1="f14526fdf81dcfc541c08f0c67bff593b7ee5ad9"/>
+		<rom name="Kids Can Read! The Night Before Christmas - Clement C. Moore LLD (USA) (En,Es).cue" size="144" crc="772af51b" sha1="b04450b5b1545a7071b4d006ea2a7c2a47600b8a"/>
+		-->
+		<description>Kids Can Read! The Night Before Christmas - Clement C. Moore LLD</description>
+		<year>1992</year>
+		<publisher>Discis Knowledge Research</publisher>
+		<info name="serial" value="25-6135 / CDRM1024580"/>
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Kids Can Read The Night Before Christmas - Clement C. Moore LLD (USA) (En,Es)" sha1="8d04cf6d3847b5c82d1b3271e3f0f3bf2cf06c51"/>
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="kids_paperbag">
+		<!--
+		Origin: redump.org
+		<rom name="Kids Can Read! The Paper Bag Princess - Robert N. Munsch (USA) (En,Es).bin" size="287797776" crc="5d6f9120" sha1="79579e11027766cdc07f6a804d06da075efb34fe"/>
+		<rom name="Kids Can Read! The Paper Bag Princess - Robert N. Munsch (USA) (En,Es).cue" size="136" crc="a25822ea" sha1="519e7a9eec0cd0476f74cf84679b0e5eb0087b62"/>
+		-->
+		<description>Kids Can Read! The Paper Bag Princess - Robert N. Munsch</description>
```

Comment:
Not sure I like this approach where original book author gets credited inside descriptions. 
It's also inconsistent across the SW itself, cfr. `R. Munsch` for Thomas Swimsuit.
Consider using a separate info XML field or a comment tag.

Reply:
I have moved the authors' names to a separate book_author info tag. For some reason Github was acting weird and the PR didn't update until I added one more commit after that, though...